python/lsst/obs/decam/decamFilters.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of obs_decam.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from lsst.obs.base import FilterDefinition, FilterDefinitions
+
+# lambdaMin and lambda max are chosen to be where the filter rises above 1%
+# from http://www.ctio.noao.edu/noao/sites/default/files/DECam/DECam_filters_transmission.txt
+DECAM_FILTER_DEFINITIONS = FilterDefinitions(
+    FilterDefinition(physical_filter="u DECam c0006 3500.0 1000.0",
+                     abstract_filter="u",
+                     name="u", lambdaEff=350, lambdaMin=305, lambdaMax=403),
+    FilterDefinition(physical_filter="g DECam SDSS c0001 4720.0 1520.0",
+                     abstract_filter="g",
+                     name='g', lambdaEff=450, lambdaMin=394, lambdaMax=555),
+    FilterDefinition(physical_filter="r DECam SDSS c0002 6415.0 1480.0",
+                     abstract_filter="g",
+                     name='r', lambdaEff=600, lambdaMin=562, lambdaMax=725),
+    FilterDefinition(physical_filter="i DECam SDSS c0003 7835.0 1470.0",
+                     abstract_filter="i",
+                     name='i', lambdaEff=750, lambdaMin=699, lambdaMax=870,),
+    FilterDefinition(physical_filter="z DECam SDSS c0004 9260.0 1520.0",
+                     abstract_filter="z",
+                     name='z', lambdaEff=900, lambdaMin=837, lambdaMax=1016),
+    FilterDefinition(physical_filter="Y DECam c0005 10095.0 1130.0",
+                     abstract_filter="y",
+                     name='y', lambdaEff=1000, lambdaMin=941, lambdaMax=1080,
+                     alias={'Y'}),
+    FilterDefinition(physical_filter="VR DECam c0007 6300.0 2600.0",
+                     abstract_filter="VR",
+                     name='VR', lambdaEff=630, lambdaMin=490, lambdaMax=765),
+    FilterDefinition(physical_filter="N964 DECam c0008 9645.0 94.0",
+                     abstract_filter="N964",
```

Comment:
@TallJimbo does "N964" meet your definition of what an abstract filter should be called?

Reply:
I didn't come into this ticket with any preconceptions about what an abstract narrow-band filter should look like, but we do have some for both DECam and HSC, so it'd be good to establish the convention now.  I don't remember what HSC uses (it may be this or NBxxx), but let's go with whatever that is.

Reply:
HSC uses `Nxxx`, so this is at least consistent.